server/serializers.py

```diff
@@ -36,7 +37,13 @@ class ModuleSerializer(serializers.ModelSerializer):
     help_url = serializers.SerializerMethodField()
 
     def get_help_url(self, obj):
-        return obj.help_url if obj.help_url else KB_ROOT_URL
+        url_pattern = re.compile('^http(?:s?)://', re.IGNORECASE)
+
+        if re.search(url_pattern, obj.help_url):
+            return obj.help_url
+
+        return f"{KB_ROOT_URL}{obj.help_url}"
```

Comment:
I had to look up 'f'. Why not just +?

Reply:
didn't realize you could do that, I haven't dug in to the full breadth of python 3 format strings